ui/src/pages/blog/[slug].tsx

```diff
@@ -187,6 +188,13 @@ const IndividualBlogPage: NextPage<Props> = (props) => {
 
     const pageTitle = `${title} - ${Config.urls.base.title}`;
 
+    // This is required to ensure that we don't encounter hydration errors when we format the published
+    // date on the client. Normally we use suppressHydrationWarning but that only works one level down,
+    // and we're passing the value to a subcomponent.
+    React.useEffect(() => {
```

Comment:
Instead of using useEffect to fix this, we can simply add an optional boolean prop called "suppressHydrationWarning" to PageSubTitle component.

Reply:
fixed